docs/open-source-tizen/tools/mic/mic-customize-image.md

```diff
@@ -1,49 +1,55 @@
-# How to Customize Images
+# Customizing Images
 
-This section will describle how to customize your image by updating the kickstart file. You can try to download a kickstart file and try to edit it. For example, download it from: 
+You can customize your image by downloading and editing the kickstart file. For more information on kickstart, see [Kickstart Options](http://fedoraproject.org/wiki/Anaconda/Kickstart).
 
-## Specify the repository
+## Specifying the Repository
 
-You can specify which repository should be used to create this package, including a local repository. For example: [http://download.tizen.org/releases/daily/trunk/ivi/latest/images/ivi-min...](http://download.tizen.org/releases/daily/trunk/ivi/latest/images/ivi-min-pc/ivi-min-pc-tizen_20120926.2.ks)
+You can specify which repository must be used to create a package, including a local repository.
+
+For example: [http://download.tizen.org/releases/daily/trunk/ivi/latest/images/ivi-min...](http://download.tizen.org/releases/daily/trunk/ivi/latest/images/ivi-min-pc/ivi-min-pc-tizen_20120926.2.ks)
```

Comment:
Link is broken

Reply:
It is legacy. As I mentioned before, mic will be obsoleted. Before then we just leave them.
Samsung doesn't take care of IVI profile.

Reply:
OK